radix-engine/src/engine/call_frame.rs

```diff
@@ -162,14 +158,15 @@ impl RENodePointer {
 
     fn borrow_native_ref<'p, 's>(
         &self, // TODO: Consider changing this to self
+        self_frame_id: usize,
```

Comment:
Minor: Isn't `self_frame_id` part of self?

Reply:
Ah no, probably named badly but `self_frame_id` is the current `frame_id` of the call frame requiring access to this node pointer. This will get cleaned up when heap is implemented.